sqpnp/sqpnp.cpp

```diff
@@ -1,13 +1,15 @@
 //
 // sqpnp.cpp
-//
-// George Terzakis (terzakig-at-hotmail-dot-com), September 2020
 // 
 // Implementation of SQPnP as described in the paper:
 //
 // "A Consistently Fast and Globally Optimal Solution to the Perspective-n-Point Problem" by G. Terzakis and M. Lourakis
 //     a) Paper:         https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123460460.pdf
 //     b) Supplementary: https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123460460-supp.pdf
+//
+// George Terzakis (terzakig-at-hotmail-dot-com), September 2020
+// Optimizations by Manolis Lourakis, February 2022, February 2024
```

Comment:
Just have both names in the comment, no need to specify what you did, unless you want to.

Reply:
It's reminder for me as the optimizations were done in two batches...